platformio.ini

```diff
@@ -52,7 +52,8 @@ build_flags =
 	${env.build_flags}
 	'-D CAMERA_MODEL_M5STACK_PSRAM'
 	'-D NAME="GROUNDLIGHT_DEMO_UNIT"'
-#	'-D ENABLE_AP'
+	'-D ENABLE_AP'
+#	'-D ENABLE_STACKLIGHT'
```

Comment:
Do the demo units no longer work with Stacklight?

Reply:
Yes we are skipping stacklight connection for now due to connection issues on my end, and once I confirmed the stacklight is functioning, I will enable it again.